cal/__init__.py

```diff
@@ -3,13 +3,45 @@
 
 from schema import db, Event, User
 from cal.fb import update_fb_events
+from celery import Celery
+
+def make_celery(app):
+    # Create the celery app.
+    celery = Celery(app.import_name, broker=app.config['CELERY_BROKER_URL'])
+    celery.conf.update(app.config)
+    TaskBase = celery.Task
+    class ContextTask(TaskBase):
+        abstract = True
+        def __call__(self, *args, **kwargs):
+            with app.app_context():
+                return TaskBase.__call__(self, *args, **kwargs)
+    celery.Task = ContextTask
+    return celery
```

Comment:
This seems overly complicated to just run functions within the `app.app_context()`. Maybe we should just manually specify `with app.app_context()` in `fb_task_test`?